moped-editor/src/views/projects/projectView/ProjectComponents/ComponentForm.js

```diff
@@ -93,54 +93,40 @@ const ComponentForm = ({
   const { component, phase } = watch();
   const subphaseOptions = useSubphaseOptions(phase?.data.moped_subphases);
   const internalTable = component?.data?.feature_layer?.internal_table;
-  const [areSignalOptionsLoaded, setAreSignalOptionsLoaded] = useState(false);
-  const onOptionsLoaded = () => setAreSignalOptionsLoaded(true);
```

Comment:
We used to track when async signal options returned from Knack because the initial values wouldn't set correctly until after the options results returned. This is no longer needed as we prep the current signal value needed for the form in the parent component and pass it as a prop.